test/Interop/Cxx/destructor/basic.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+
+import Basic
+import StdlibUnittest
+
+var CXXDestructorTestSuite = TestSuite("CXX Destructor")
+
+@inline(never)
+func createTypeWithUserProvidedDestructor(with ptr: UnsafeMutablePointer<Int32>) {
```

Comment:
Please don't use "with" as an argument label. It is not meaningful (because it can be added to every constructor argument). No label would be better here and below.